README.md

```diff
@@ -546,6 +547,28 @@ Additionally an optional string parameter `system_prompt` can be specified in `/
 
 Activate token quota limits for the service by adding a new configuration structure into the configuration file. That structure should be added into `ols_config` section. It has the following format:
```

Comment:
Apologies for being a pest :), but this paragraph seems out of place. Could we just remove it?

Reply:
fixed, moved to the right location